libs/api/prisma/migrations/20240723095847_add_currency_to_memberplan/migration.sql

```diff
@@ -0,0 +1,9 @@
+-- CreateEnum
+CREATE TYPE "Currency" AS ENUM ('CHF', 'EUR');
+
+-- AlterTable
+ALTER TABLE "member.plans" ADD COLUMN "currency" "Currency" NOT NULL DEFAULT 'CHF';
+
+-- AlterTable
+ALTER TABLE "member.plans" ALTER COLUMN "currency" DROP DEFAULT;
```

Comment:
Also add currency to the subscription and invoice table and copy it on subscription creation. This is important to ensure that when a admin changes the currency on member plan level, the currency says the same for existing subscriptions and invoices to keep the record clean. Also you can then remove a lot of database queries you added in the code to get the member plan because the subscription is already available.